clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -1,19 +1,18 @@
 {
-    "name": "vscode-languageserver-extension",
+    "name": "LSP for COBOL",
```

Comment:
The name should be the id of extension. 
name: lsp-for-cobol.
displayName field should have a human-readable extension name.
Also please provide text for description.
Please also check value for the author

Reply:
Hey, i agree with Iurii, however i don't currently have the access to apply the changes.
Display Name: LSP for COBOL
Description:
The LSP for COBOL extension standardizes the communication between language tooling and your code editor.
Author would seem to be "Broadcom"